DocIndexRetriever/docker/xeon/docker_compose.yaml

```diff
@@ -9,34 +9,38 @@ services:
     image: redis/redis-stack:7.2.0-v9
     container_name: redis-vector-db
     ports:
-      - "16379:6379"
+      - "6379:6379"
       - "8001:8001"
   dataprep-redis-service:
-    image: opea/dataprep-on-ray-redis:latest
+    image: opea/dataprep-redis:latest
     container_name: dataprep-redis-server
     depends_on:
       - redis-vector-db
     ports:
       - "6007:6007"
+      - "6008:6008"
+      - "6009:6009"
```

Comment:
why we need 6008 and 6009?

Reply:
They added two endpoints for get and delete docs